contracts/dependency-registry/Scripty.sol

```diff
@@ -0,0 +1,4 @@
+// SPDX-License-Identifier: Unlicense
```

Comment:
Added this file so that hardhat would compile ScriptyBuilder for testing. @jakerockland @ryley-o let me know if there's a better way to do this

Reply:
we sometimes put those files in `/contracts/libs/integration-refs/<reference-name>/`, just to clearly designate them as references that we didn't design

Reply:
probably worth adding a dev comment to the file indicating what you did in this comment?